includes/definitions/discovery/vrp.yaml

```diff
@@ -336,16 +344,14 @@ modules:
                     descr: '{{ $entPhysicalName }} Rx'
                     entPhysicalIndex: '{{ $index }}'
                     entPhysicalIndex_measured: ports
-                    index: 'lane-rx-{{ $index }}'
+                    index: 'rx-{{ $index }}'
                     divisor: 100
                     group: '{{ $entPhysicalName }}'
                     high_limit: hwEntityOpticalRxHighThreshold
+                    warn_limit: hwEntityOpticalRxHighWarnThreshold
+                    low_warn_limit: hwEntityOpticalRxLowWarnThreshold
                     low_limit: hwEntityOpticalRxLowThreshold
                     skip_values:
-                        -
-                            oid: hwEntityOpticalRxPower
-                            op: '>'
-                            value: 0
```

Comment:
This check was a way to check if the OpticalRxPower was provided in mW or dbm ... If you remove this, it will break half of the devices.